pep-0565.rst

```diff
@@ -347,32 +347,21 @@ References
    (https://github.com/python/cpython/pull/4458)
 
 .. [5] Tracker issue for PEP 565 implementation
-   (https://bugs.python.org/issue31975)
+   (https://github.com/python/cpython/issues/76156)
 
-.. [6] First python-dev discussion thread
-   (https://mail.python.org/pipermail/python-dev/2017-November/150477.html)
+.. [6] IPython's ``DeprecationWarning`` auto-configuration
+   (https://github.com/ipython/ipython/blob/6.2.x/IPython/core/interactiveshell.py#L619)
 
-.. [7] Second python-dev discussion thread
-   (https://mail.python.org/pipermail/python-dev/2017-November/150819.html)
+.. [7] ``warnings.hide_warnings`` API proposal
+   (https://github.com/python/cpython/issues/76410)
 
-.. [8] IPython's DeprecationWarning auto-configuration
-   (https://github.com/ipython/ipython/blob/6.2.x/IPython/core/interactiveshell.py#L619)
+* First python-dev discussion thread
+   (https://mail.python.org/pipermail/python-dev/2017-November/150477.html)
```

Comment:
May as well make this an actual link, no?

```suggestion
* `First python-dev discussion thread
  <https://mail.python.org/pipermail/python-dev/2017-November/150477.html>`__
```

Reply:
Yep, good idea, x2